src/services/email.services.js

```diff
@@ -0,0 +1,88 @@
+import nodemailer from 'nodemailer';
+import 'dotenv/config';
+import jwt from 'jsonwebtoken';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
```

Comment:
It would be beneficial to handle potential errors when creating the transporter, such as missing environment variables or incorrect SMTP configuration.